### PR TITLE
Minor build instructions added

### PR DESCRIPTION
**Pull request by BarmenduC**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,4 +7,15 @@ You can create, edit and delete events, and cancel or hide them, categorize even
...
 
+Requirements
+-------
+To build Calendar you need some development libraries:
+
+* sqlite3   -- ```$ pkgman install sqlite_devel```
+
+Note
+-------
+Calendar can be built on 64-bit version of Haiku only.
```
Comment (humdingerb) [inline README.md:18]: Really? I thought you just need to change to gcc11+ with "setarch x86" for 32bit Haiku.

Comment (BarmenduC) [reply README.md:18]: I tried the setarch x86, but it throws up a bunch of errors while running 'make'.

Any suggestions on how to avoid that?

Comment (humdingerb) [reply README.md:18]: What are those errors?

Comment (BarmenduC) [reply README.md:18]: Let me put them down for you 


g++ -c BeCalc.cpp -iquote./  -iquote./   -O3    -o "objects.x86-cc8-release/BeCalc.o"
In file included from BeCalcApp.h:5,
                 from BeCalc.cpp:1:
BeCalcWin.h:96:3: error: 'BFilePanel' does not name a type
   BFilePanel *save;
   ^~~~~~~~~~
BeCalc.cpp: In function 'int main()':
BeCalc.cpp:24:45: warning: ISO C++ forbids converting a string constant to 'char*' [-Wwrite-strings]
  BeCalcApp app("application/x-vnd.CI-BeCalc");
                                             ^
BeCalc.cpp:54:54: warning: ISO C++ forbids converting a string constant to 'char*' [-Wwrite-strings]
  app.window=new BeCalcWin(BECALCNAME, B_TITLED_WINDOW);
                                                      ^
/boot/system/develop/etc/makefile-engine:300: recipe for target 'objects.x86-cc8-release/BeCalc.o' failed
make: *** [objects.x86-cc8-release/BeCalc.o] Error 1


Do tell me if you have some suggestions and I'll try them out and modify the pull request according to that.

Thanks!!

Comment (humdingerb) [reply README.md:18]: It looks like you need to link to libtracker for the above.
Only, this is about the Calendar app, not BeCalc...

File: README.md [hunk 2]
```diff
@@ -7,4 +7,15 @@ You can create, edit and delete events, and cancel or hide them, categorize even
 
 ![screenshot](documentation/images/main_window.png)
 
+Requirements
+-------
+To build Calendar you need some development libraries:
+
+* sqlite3   -- ```$ pkgman install sqlite_devel```
```
Comment (humdingerb) [inline README.md:14]: For 32bit Haiku, that'd be `pkgman install sqlite_x86_devel` as it'll only build with a newer gcc, not the old gcc2 Haiku 32bit defaults to.

Comment (BarmenduC) [reply README.md:14]: Okay... so that's it.

Should I try it out and if it works then modify the instructions in the pull request?

Comment (BarmenduC) [reply README.md]: I've modified the instructions @humdingerb .

File: README.md [hunk 3]
```diff
@@ -7,4 +7,20 @@ You can create, edit and delete events, and cancel or hide them, categorize even
 
 ![screenshot](documentation/images/main_window.png)
 
+Requirements
+-------
+To build Calendar you need some development libraries:
```
Comment (humdingerb) [inline README.md:12]: We can be even more specific:
```suggestion
To build Calendar you need the sqlite3 development library:
```

File: README.md [hunk 4]
```diff
@@ -7,4 +7,20 @@ You can create, edit and delete events, and cancel or hide them, categorize even
...
+
+* sqlite3
```
Comment (humdingerb) [inline README.md:14]: Can then be removed.

File: README.md [hunk 5]
```diff
@@ -7,4 +7,20 @@ You can create, edit and delete events, and cancel or hide them, categorize even
...
+* sqlite3   
+
+  For 32 bit version of Haiku   -- ```$ pkgman install sqlite_x86_devel```
+  
+  For 64 bit version of Haiku   -- ```$ pkgman install sqlite_devel```  
+   
+
+Note
+-------
+Before invoking 'make' (for 32 bit version of Haiku), do a 'setarch x86' to change the build environment to use gcc11.
```
Comment (humdingerb) [inline README.md:23]:
```suggestion
When building on 32bit Haiku, you need to change the build environment to use gcc11 with `setarch x86` before invoking `make`. 
```

Comment (BarmenduC) [reply README.md]: All suggested changes have been made @humdingerb .

**Comment by humdingerb** [comment]
Thanks!

**Pull request merged**